addons/resource.language.en_gb/resources/strings.po

```diff
@@ -21955,3 +21955,15 @@ msgstr ""
 msgctxt "#39118"
 msgid "Share debug log"
 msgstr ""
+
+#. Label for enabling scaling filter for better Pixel Art Games
+#:xbmc/settings/settings.xml
+msgctxt "#39119"
+msgid "Enable Integer Scaling Filter (NN)"
+msgstr ""
+
+#. Description of setting with label #39119 "Enable Integer Scaling Filter (NN)"
+#: system/settings/settings.xml
+msgctxt "#39120"
+msgid "Integer scaling (IS) is a nearest-neighbor upscaling technique that simply scales up the existing pixels by an integer (i.e., whole number) multiplier.Nearest-neighbor (NN) interpolation works by filling in the missing color values in the upscaled image with that of the coordinate-mapped nearest source pixel value"
```

Comment:
I would propose a slight simplification (and space between sentences) of the help text.

Before:

> Integer scaling (IS) is a nearest-neighbor upscaling technique that simply scales up the existing pixels by an integer (i.e., whole number) multiplier.Nearest-neighbor (NN) interpolation works by filling in the missing color values in the upscaled image with that of the coordinate-mapped nearest source pixel value"

After:

> Integer scaling (IS) is an upscaling technique that scales up the existing pixels by a whole number. Nearest-neighbor (NN) interpolation works by filling in the missing color values with that of the nearest source pixel value.